LPD-52573 move to playright

ZakiMiklos opened the pull request.

shuyangzhou committed:
LPD-55206 Make sure springboot test dependencies is using origin spring-test jar, rather than jakarta patched version from tooling classpath.

BryanEngler committed:
LPD-55006 portal-search-elasticsearch7-impl: Make it easier to debug checksum mismatch

https://liferay.atlassian.net/browse/LPD-55006

dantewang committed:
LPD-52892 Inline the prefs into getter methods, and remove CompanySecurityBag

dantewang committed:
LPD-52892 Regenerated

abesousa committed:
LPD-24056 Add column system to ListTypeEntry

abesousa committed:
LPD-24056 buildService

abesousa committed:
LPD-24056 Add upgrade process to populate the new column

abesousa committed:
LPD-24056 Add upgrade process test

abesousa committed:
LPD-24056 Add parameter system to service and local service

# breaking

## What modules/apps/list-type/list-type-api/src/main/java/com/liferay/list/type/service/ListTypeEntryService.java

The method addListTypeEntry from ListTypeEntryService now requires an additional parameter: boolean system.

## Why

We need this parameter to differentiate system and custom picklist items.
That way we can edit only the custom entries added to a system picklist.

----

abesousa committed:
LPD-24056 buildService

abesousa committed:
LPD-24056 Add logic of system picklist entries under FF

abesousa committed:
LPD-24056 Add service tests

abesousa committed:
LPD-24056 Add system property to ListTypeEntry

abesousa committed:
LPD-24056 Add logic of system picklist entries to resource layer

abesousa committed:
LPD-24056 Add system attribute to existing system picklists

abesousa committed:
LPD-24056 Adapt UI to system picklists entries

dantewang committed:
LPD-53805 portlet-tck-bridge: include all cookie names appeared in TCK tests

cdbm committed:
LPD-41009 Use the proper company context

cdbm committed:
LPD-41009 fix AnnotatedApplicationClientTest for database partition enabled

cdbm committed:
LPD-41009 fix JWTAssertAuthorizationGrantTest for database partition enabled

cdbm committed:
LPD-41009 fix AnnotatedApplicationPrefixHandlerClientTest for database partition enabled

dantewang committed:
LPD-53806 Fix V2ResponseTests_PortletResponse_ApiResource_addPropertyA2: Method addProperty(javax.servlet.http.Cookie): Throws IllegalArgumentException if the specified cookie is null.

dantewang committed:
LPD-53804 Bring back the logic to download chromedriver and apply

abesousa committed:
LPD-24056 Add resource tests

abesousa committed:
LPD-24056 buildREST

cdbm committed:
LPD-41009 fix AnnotationsAndHttpApplicationClientTest for database partition enabled

cdbm committed:
LPD-41009 fix AnnotationsAndHttpPrefixApplicationClientTest for database partition enabled

cdbm committed:
LPD-41009 fix CORSApplicationClientTest for database partition enabled

cdbm committed:
LPD-41009 fix DisabledUserClientTest for database partition enabled

cdbm committed:
LPD-41009 fix ExpiredAuthorizationsAfterlifeTest for database partition enabled

cdbm committed:
LPD-41009 fix GrantAuthorizationCodeKillSwitchTest for database partition enabled

cdbm committed:
LPD-41009 fix GrantAuthorizationCodePKCEKillSwitchTest for database partition enabled

abesousa committed:
LPD-24056 Fix compile errors due to additional parameter in addListTypeEntry of LocalService

gislayne-vitorino committed:
LPD-53948 Add constants for new publication roles

gislayne-vitorino committed:
LPD-53948 Add and update descriptions for Publications roles

cdbm committed:
LPD-41009 fix GrantClientKillSwitchTest for database partition enabled

cdbm committed:
LPD-41009 fix GrantedFlowsTest for database partition enabled

cdbm committed:
LPD-41009 fix GrantResourceOwnerKillSwitchTest for database partition enabled

cdbm committed:
LPD-41009 fix HttpMethodApplicationClientTest for database partition enabled

cdbm committed:
LPD-41009 fix NarrowDownScopeClientTest for database partition enabled

cdbm committed:
LPD-41009 fix OAuth2AuthorizationClientTest for database partition enabled

cdbm committed:
LPD-41009 fix OAuth2WebServerServletTest for database partition enabled

cdbm committed:
LPD-41009 fix RememberDeviceApplicationClientTest for database partition enabled

cdbm committed:
LPD-41009 fix SAPClientTest for database partition enabled

cdbm committed:
LPD-41009 fix ScopeCheckerGuestAllowedTest for database partition enabled

cdbm committed:
LPD-41009 fix ScopeFinderTest for database partition enabled

cdbm committed:
LPD-41009 fix ScopeMapperNarrowDownClientTest for database partition enabled

cdbm committed:
LPD-41009 fix SecurityTest for database partition enabled

cdbm committed:
LPD-41009 fix TOCTOUTest for database partition enabled

cdbm committed:
LPD-41009 fix TokenExpeditionTest for database partition enabled

cdbm committed:
LPD-41009 fix TokenIntrospectionTest for database partition enabled

dantewang committed:
LPD-53804 Add Chrome logic for CI

cdbm committed:
LPD-41009 fix TrustedApplicationClientTest for database partition enabled

cdbm committed:
LPD-41009 fix RefreshTokenAuthorizationGrantTest for database partition enabled

dantewang committed:
LPD-53804 Add portal properties to make selenium work with chrome

ZakiMiklos committed:
LPD-53901 Move to playwright

gislayne-vitorino committed:
LPD-53948 Add publication roles

cdbm committed:
LPD-41009 fix IssueJWTAccessTokenTest for databse partition enabled

tinatian committed:
LPD-53804 Extract common logic

abesousa committed:
LPD-24056 SF

tinatian committed:
LPD-55212 Fix V2EnvironmentTests, we no longer see the exception after removing the logic added by LPS-76502 which was used to fix tck test failures.

gislayne-vitorino committed:
LPD-53948 Add Publications User role automatically after one of the new Publications role is added

cdbm committed:
LPD-41009 fix IsolationAcrossCompaniesTest for databse partition enabled

cdbm committed:
LPD-41009 fix JsonWebServiceTest for databse partition enabled

AliciaGarciaGarcia committed:
LPD-54305 add taxonomyCategoryExternalReferenceCode, parentTaxonomyCategory and parentTaxonomyVocabulary to TaxonomyCategoryBrief to allow lazy references loading.
parentTaxonomyCategory and parentTaxonomyVocabulary will be write only

AliciaGarciaGarcia committed:
LPD-54305 we need TaxonomyCategoryBrief not to be read only

AliciaGarciaGarcia committed:
LPD-54305 buildRest

DavysonMelo committed:
LPD-53697 Remove inline styles portal rules engine sample web

AliciaGarciaGarcia committed:
LPD-54305 show the TaxonomyCategoryExternalReferenceCode field on the TaxonomyCategoryBrief

AliciaGarciaGarcia committed:
LPD-54305 fix tests

gislayne-vitorino committed:
LPD-53948 Add and use Util to avoid code duplication

cdbm committed:
LPD-41009 fix LocalOAuthClientTest for databse partition enabled

cdbm committed:
LPD-41009 SF

abesousa committed:
LPD-24056 baseline

wanderlast committed:
LPD-54453 - send notification only if user is already approved

wanderlast committed:
LPD-54453 - add test

wanderlast committed:
LPD-54453 - SF

wanderlast committed:
LPD-54453 - remove extra notification call

wanderlast committed:
LPD-54453 - fix test to fit new behavior

rachaelkoestartyo committed:
LPD-53971 implement getOverviewContent

tinatian committed:
LPD-53804 Set session timeout to 3600

pedro-oliveira446 committed:
LPD-54604 Remove the definitions after their usage is complete

algobob committed:
LPD-53553 [Test Fix] portal-workflow-kaleo-designer-web/actions.spec.ts > can save a workflow definition that has a customer extension action after changing from an Update

david-truong committed:
LPD-54533 build lang

david-truong committed:
LPD-54533 add test for default owner action ids configuration

david-truong committed:
LPD-54533 don't convert null values to "null"

david-truong committed:
LPD-54533 add configuration to change owner default permissions

# breaking

## What modules/apps/change-tracking/change-tracking-api/src/main/java/com/liferay/change/tracking/configuration/CTSettingsConfiguration.java

Added a new configurations

## Why

To allow users to change the default owner permissions

----

# breaking

## What modules/apps/change-tracking/change-tracking-api/src/main/java/com/liferay/change/tracking/configuration/CTConflictConfiguration.java

Deleted the class

## Why

Combined with CTSettingsConfiguration to allow
CTCollectionLocalServiceImpl to access required configurations

----

Mikellorza committed:
LPD-53915 Make thumbnailURL nested field if requested

movaldivia committed:
LPD-54468 a json patch should use PUT in the generated PATCH only if the PUT has json impl. Not when only has multipart

movaldivia committed:
LPD-54468 add simple test entity to showcase

movaldivia committed:
LPD-54468 buildRest

Mikellorza committed:
LPD-53915 Update test

movaldivia committed:
LPD-54468 Auto SF

ambrinchaudhary committed:
LPD-54028 Create service to update folders

ambrinchaudhary committed:
LPD-54028 Call api action when onSubmit

ambrinchaudhary committed:
LPD-54028 Show result with a toast message

ambrinchaudhary committed:
LPD-54028 Disable button while submitting

david-truong committed:
LPD-54533 ant update-portal-osgi-configuration-properties

cdbm committed:
LPD-41009 fix DenyAccessToAdminScopeTononAdminUserTest for database partition enabled

cdbm committed:
LPD-41009 fix RevokedTokenTest for database partition enabled

carolmariaabb committed:
LPD-24056 Adjust tests

thiagobuarque-liferay committed:
LPD-54702 compare by the panel entry key when two panel categories have the same order and key

boton committed:
LPD-54566 Add test to files section download action

thiagobuarque-liferay committed:
LPD-54702 update existing panel categories that have same order and category key

cdbm committed:
LPD-41009 use Exception

yuriymr committed:
LPD-54428 Using the ProviderType annotation on WorkflowMetricsReindexerRegistry to fix Workflow Metrics

yuriymr committed:
LPD-54428 Adjusting test

thiagobuarque-liferay committed:
LPD-54702 add test to assert warning

dsanz committed:
LPD-54810 Migrate additional API URL parameters to system data set advanced sample

dsanz committed:
LPD-54810 Migrate items actions to system data set advanced sample

dsanz committed:
LPD-54810 Migrate bulk actions to system data set advanced sample

dsanz committed:
LPD-54810 Migrate sorts to system data set advanced sample

abfranca committed:
LRSD-7795 Fix selected tickets on Edit page

daiazius committed:
LPD-52902 Changing the configuration wasn't updating the fields in SimpleCaptchaImpl. Refactor the class to remove the fields and render using the updated configuration.

# breaking

## What modules/apps/captcha/captcha-api/src/main/java/com/liferay/captcha/simplecaptcha/SimpleCaptchaImpl.java

Removed the activate and init methods that were used to configure the captcha based on the settings.

## Why

This was required to fix a bug that made captcha instance settings only update on restart.

----

dsanz committed:
LPD-54810 Make items actions importable by DSM

boton committed:
LPD-54566 Add deleteObjectEntry to delete object entries by ID

boton committed:
LPD-54566 Add clean up

movaldivia committed:
LPD-54468 manage both cases xml and json with helper function

movaldivia committed:
LPD-54468 SF

gislayne-vitorino committed:
LPD-53948 Add test

movaldivia committed:
LPD-54468 Auto SF

igor-franca committed:
LPD-54871 Include form playwright tests in the BPM routine

michaelhashimoto committed:
LRCI-5789 Set user.home if it is not already set

michaelhashimoto committed:
LRCI-5789 Always create an ref file when interacting with s3

michaelhashimoto committed:
LRCI-5789 Replace S3 Object path refs when referenced in copy methods

igor-franca committed:
LPD-54871 Remove unnecessary aria tag to avoid double label reference

michaelhashimoto committed:
LRCI-5789 Add method to check if s3 object ref is available

CharlotteFrancis committed:
LRCI-5807 add priority attribute to ReinvokeRule class

rafaprax committed:
LPD-41009 SF

jonathanmccann committed:
LPD-54933 Reduce flakiness for memberships.spec.ts

velosofelipe64 committed:
LRSD-9296 remove SV name link

kevenleone committed:
LPD-54963 Update Types

kevenleone committed:
LPD-54963 Simplify DashboardNavigation

kevenleone committed:
LPD-54963 Create ListView component

kevenleone committed:
LPD-54963 Rename Solution folder

kevenleone committed:
LPD-54963 Update entities and constants

kevenleone committed:
LPD-54963 Add new Administrator Pages, refactor some pages and delete some code

kevenleone committed:
LPD-54963 Update import names

kevenleone committed:
LPD-54963 Update base components

kevenleone committed:
LPD-54963 Delete unused, add new outletContext export

kevenleone committed:
LPD-54963 No longer request the Channel data inside MarketplaceContext, parse marketplaceProduct after getProducts API is called

jonathanmccann committed:
LPD-54933 Use existing helper methods and increase time outs to avoid flakiness

jennyhchen committed:
LRSD-7216 LRSD-9495 Convert NaN MM-dd-yyyy date to MM/dd/yyyy for Safari validation

michaelhashimoto committed:
LRCI-5789 Allow for recursively referenced s3 object paths

tinatian committed:
LPD-52892 SF, inline private method logic and simplify

CharlotteFrancis committed:
LRCI-5807 use priority attribute to sort ReinvokeRules

CharlotteFrancis committed:
LRCI-5807 add a default priority for reinvoke rules

CharlotteFrancis committed:
LRCI-5807 SF

david-truong committed:
LPD-54533 add upgrade process to move configurations

david-truong committed:
LPD-54533 update tests to use CTSettingsConfiguration instead of CTConflictConfiguration

david-truong committed:
LPD-54533 add upgrade test

ChrisKian committed:
LPD-46627 The MembershipsUserFieldExpressionHandler class expects the 'userGroups' attribute to be prepended with 'membership:', however explicitly doing so in the SP connection will cause problems with the default attribute resolver.  Instead, make sure we add 'membership:' to the attribute name before publishing."

ChrisKian committed:
LPD-46627 If the UserGroup does not yet exist, create it first

Mikellorza committed:
LPD-53915 Add thumbnail url to the nested fields request

gislayne-vitorino committed:
LPD-53948 Update test

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

ZakiMiklos commented:
ci:test:relevant

ZakiMiklos commented:
ci:test:headless-playwright

ZakiMiklos commented:
ci:test:headless-playwright

ccorreagg committed:
LPD-55089 Update to the newer version

yuriymr committed:
LPD-54428 baseline

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>headless-playwright</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest(master)/14426/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-40.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/14426/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>relevant</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest(master)/6821/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-35.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/6821/jenkins-report.html">here</a>.</b></p>

JhossephG committed:
LPD-55090 Pass companyId to FeatureFlagManagerUtil

liferay-continuous-integration commented:
<html><h3>:x: ci:test:relevant - 0 out of 1 jobs passed in 2 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/387d567711c76d628a32704dffacf46e974102ce">387d567711c76d628a32704dffacf46e974102ce</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/387d567711c76d628a32704dffacf46e974102ce">387d567711c76d628a32704dffacf46e974102ce</a><br/>Jenkins Build URL: <a href="https://testray.liferay.com/#/project/35392/routines/590307/build/210683781">EE Development Acceptance (master) - 1346 - 2025-05-06[20:34:36]</a></p><details><summary><strong>ci:test:relevant - 0 out of 1 jobs PASSED</strong></summary><div><h4>1 Failed Jobs:</h4><ul><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest(master)/6821/"><strike>test-portal-acceptance-pullrequest(master)</strike></a></li></ul></div></details><h5>For more details click <a href="https://test-1-35.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/6821/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><h5><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest(master)/6821/">test-portal-acceptance-pullrequest(master)</a></h5><div><h6>Job Results:</h6><p>0 Jobs Passed.<br/>1 Job Failed.</p></div><div><p>The sender branch SHA could not be found on <strong><a href="https://github.com/ZakiMiklos/liferay-portal/pull/8">ZakiMiklos/LPD-52573-move-to-playwright</a></strong>. The sender branch may have been force pushed or deleted after the pull request test was initiated.</p><pre>fatal: Not a valid branch point: 'c102a37de76bbb57653884d5747c8a392e9df817'.</pre></div></div></li></ol><h4>For upstream results, click <a href="https://test-1-40.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">here</a>.</h4></details></html>

ZakiMiklos commented:
ci:test:relevant

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest(master)/6821/">test-portal-acceptance-pullrequest(master)#6821</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-35.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/6821/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>relevant<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/ZakiMiklos/liferay-portal/pull/8">ZakiMiklos#8</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357/build/210939506">[master] ci:test:relevant - ZakiMiklos &gt; ZakiMiklos - PR#8 - 2025-05-07[04:40:02]</a><br/></div><div><strong>Testray Build ID:</strong><span></span><code>210939506</code><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-35.liferay.com/job/publish-testray-report/14947/">publish-testray-report#14947</a><br/></div></div>

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>relevant</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest(master)/22235/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-37.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/22235/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<h1>Build completed.</h1><p>Jenkins URL: <a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest(master)/22235/">test-portal-acceptance-pullrequest(master)</a></p><p>The <code>relevant</code> test suite did not execute tests because all modified files were excluded by the <b>modified.files.global.excludes</b> property in <code>test.properties</code>.</p><p>Please use <code>ci:test:sf</code> and/or <code>ci:forward</code> for this pull request to be forwarded or eligible for forwarding.</p>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest(master)/22235/">test-portal-acceptance-pullrequest(master)#22235</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-37.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/22235/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>relevant<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/ZakiMiklos/liferay-portal/pull/8">ZakiMiklos#8</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357/build/210939518">[master] ci:test:relevant - ZakiMiklos &gt; ZakiMiklos - PR#8 - 2025-05-07[04:45:24]</a><br/></div><div><strong>Testray Build ID:</strong><span></span><code>210939518</code><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-37.liferay.com/job/publish-testray-report/47695/">publish-testray-report#47695</a><br/></div></div>

lfbesada committed:
LPD-55091 Add asserts to ensure the tag style is added only if necessary

lfbesada committed:
LPD-55091 Add the test info annotations

liferay-continuous-integration commented:
<html><h3>:x: ci:test:headless-playwright - 6 out of 9 jobs passed in 51 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/387d567711c76d628a32704dffacf46e974102ce">387d567711c76d628a32704dffacf46e974102ce</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/387d567711c76d628a32704dffacf46e974102ce">387d567711c76d628a32704dffacf46e974102ce</a><br/>Jenkins Build URL: <a href="https://testray.liferay.com/#/project/35392/routines/590307/build/210683781">EE Development Acceptance (master) - 1346 - 2025-05-06[20:34:36]</a></p><details><summary><strong>ci:test:headless-playwright - 6 out of 9 jobs PASSED</strong></summary><div><h4>3 Failed Jobs:</h4><ul><li><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest(master)/14426/"><strike>test-portal-acceptance-pullrequest(master)</strike></a></li><li>playwright-js-tomcat90-postgresql163<ul><li><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/918011/">playwright-js-tomcat90-postgresql163/0/0</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1205952/"><strike>playwright-js-tomcat90-postgresql163/1/0</strike></a></li><li><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/918012/">playwright-js-tomcat90-postgresql163/2/0</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1241884/">playwright-js-tomcat90-postgresql163/3/0</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1205953/"><strike>playwright-js-tomcat90-postgresql163/4/0</strike></a></li><li><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/918013/">playwright-js-tomcat90-postgresql163/5/0</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1400220/">playwright-js-tomcat90-postgresql163/6/0</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1400219/">playwright-js-tomcat90-postgresql163/7/0</a></li></ul></li></ul></div><details><summary><strong>6 Successful Jobs:</strong></summary><ul/></details></details><h5>For more details click <a href="https://test-1-40.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/14426/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1205952//consoleText">playwright-js-tomcat90-postgresql163/1/0</a> - <a href="https://storage.cloud.google.com/testray-results/2025-05/test-1-40/test-portal-acceptance-pullrequest(master)/14426/playwright-js-tomcat90-postgresql163/1/0/playwright-report/index.html">Playwright Report</a><ol><li><details><summary><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1205952/testReport/export-import-web_main_stagingUseCase.spec/ts">export-import-web/main/stagingUseCase.spec.ts</a></summary><div><ol><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1205952//testReport/export-import-web_main_stagingUseCase.spec/ts/Staging_publish_template_smoke">Staging publish template smoke</a></li></ol></div></details></li></ol></div></li></ol><hr/><details><summary><strong>Failures in common with <a href="https://test-1-40.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">acceptance upstream results</a> at 387d567711c76d628a32704dffacf46e974102ce:</strong></summary><ol><li><div><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1205952//consoleText">playwright-js-tomcat90-postgresql163/1/0</a> - <a href="https://storage.cloud.google.com/testray-results/2025-05/test-1-40/test-portal-acceptance-pullrequest(master)/14426/playwright-js-tomcat90-postgresql163/1/0/playwright-report/index.html">Playwright Report</a><ol><li><details><summary><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1205952/testReport/export-import-web_main_instance.export.spec/ts">export-import-web/main/instance.export.spec.ts</a></summary><div><ol><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1205952//testReport/export-import-web_main_instance.export.spec/ts/can_export_custom_object_entries_at_instance_level_with_date_filter">can export custom object entries at instance level with date filter</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1205952//testReport/export-import-web_main_instance.export.spec/ts/can_export_new_default_and_custom_task_name">can export new default and custom task name</a></li></ol></div></details></li></ol></div></li><li><div><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1205953//consoleText">playwright-js-tomcat90-postgresql163/4/0</a> - <a href="https://storage.cloud.google.com/testray-results/2025-05/test-1-40/test-portal-acceptance-pullrequest(master)/14426/playwright-js-tomcat90-postgresql163/4/0/playwright-report/index.html">Playwright Report</a><ol><li><details><summary><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1205953/testReport/batch-planner_main_export.spec/ts">batch-planner/main/export.spec.ts</a></summary><div><ol><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1205953//testReport/batch-planner_main_export.spec/ts/can_export_as_JSON_with_all_field_types_mapped">can export as JSON with all field types mapped</a></li></ol></div></details></li></ol></div></li></ol></details></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest(master)/14426/">test-portal-acceptance-pullrequest(master)#14426</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-40.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/14426/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>headless-playwright<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/ZakiMiklos/liferay-portal/pull/8">ZakiMiklos#8</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357/build/210955103">[master] ci:test:headless-playwright - ZakiMiklos &gt; ZakiMiklos - PR#8 - 2025-05-07[04:38:26]</a><br/></div><div><strong>Testray Build ID:</strong><span></span><code>210955103</code><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-40.liferay.com/job/publish-testray-report/31208/">publish-testray-report#31208</a><br/></div></div>

ZakiMiklos commented:
ci:test:headless-playwright

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>headless-playwright</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest(master)/22238/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-37.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/22238/jenkins-report.html">here</a>.</b></p>

lfbesada committed:
LPD-55091 Prevent from adding empty style tags on rows and containers

lfbesada committed:
LPD-55091 Prevent from adding empty style tags on collections

lfbesada committed:
LPD-55091 Prevent from adding empty style tags on forms

lfbesada committed:
LPD-55091 Prevent from adding empty style tags on fragments

liferay-continuous-integration commented:
<html><h3>:x: ci:test:headless-playwright - 6 out of 9 jobs passed in 52 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/387d567711c76d628a32704dffacf46e974102ce">387d567711c76d628a32704dffacf46e974102ce</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/387d567711c76d628a32704dffacf46e974102ce">387d567711c76d628a32704dffacf46e974102ce</a><br/>Jenkins Build URL: <a href="https://testray.liferay.com/#/project/35392/routines/590307/build/210683781">EE Development Acceptance (master) - 1346 - 2025-05-06[20:34:36]</a></p><details><summary><strong>ci:test:headless-playwright - 6 out of 9 jobs PASSED</strong></summary><div><h4>3 Failed Jobs:</h4><ul><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest(master)/22238/"><strike>test-portal-acceptance-pullrequest(master)</strike></a></li><li>playwright-js-tomcat90-postgresql163<ul><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1400289/">playwright-js-tomcat90-postgresql163/0/0</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1260568/"><strike>playwright-js-tomcat90-postgresql163/1/0</strike></a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1155875/">playwright-js-tomcat90-postgresql163/2/0</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1400290/">playwright-js-tomcat90-postgresql163/3/0</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1260567/"><strike>playwright-js-tomcat90-postgresql163/4/0</strike></a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1155876/">playwright-js-tomcat90-postgresql163/5/0</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1206015/">playwright-js-tomcat90-postgresql163/6/0</a></li><li><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/918077/">playwright-js-tomcat90-postgresql163/7/0</a></li></ul></li></ul></div><details><summary><strong>6 Successful Jobs:</strong></summary><ul/></details></details><h5>For more details click <a href="https://test-1-37.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/22238/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1260568//consoleText">playwright-js-tomcat90-postgresql163/1/0</a> - <a href="https://storage.cloud.google.com/testray-results/2025-05/test-1-37/test-portal-acceptance-pullrequest(master)/22238/playwright-js-tomcat90-postgresql163/1/0/playwright-report/index.html">Playwright Report</a><ol><li><details><summary><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1260568/testReport/export-import-web_main_stagingUseCase.spec/ts">export-import-web/main/stagingUseCase.spec.ts</a></summary><div><ol><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1260568//testReport/export-import-web_main_stagingUseCase.spec/ts/Staging_publish_template_smoke">Staging publish template smoke</a></li></ol></div></details></li></ol></div></li></ol><hr/><details><summary><strong>Failures in common with <a href="https://test-1-40.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">acceptance upstream results</a> at 387d567711c76d628a32704dffacf46e974102ce:</strong></summary><ol><li><div><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1260568//consoleText">playwright-js-tomcat90-postgresql163/1/0</a> - <a href="https://storage.cloud.google.com/testray-results/2025-05/test-1-37/test-portal-acceptance-pullrequest(master)/22238/playwright-js-tomcat90-postgresql163/1/0/playwright-report/index.html">Playwright Report</a><ol><li><details><summary><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1260568/testReport/export-import-web_main_instance.export.spec/ts">export-import-web/main/instance.export.spec.ts</a></summary><div><ol><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1260568//testReport/export-import-web_main_instance.export.spec/ts/can_export_custom_object_entries_at_instance_level_with_date_filter">can export custom object entries at instance level with date filter</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1260568//testReport/export-import-web_main_instance.export.spec/ts/can_export_new_default_and_custom_task_name">can export new default and custom task name</a></li></ol></div></details></li></ol></div></li><li><div><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1260567//consoleText">playwright-js-tomcat90-postgresql163/4/0</a> - <a href="https://storage.cloud.google.com/testray-results/2025-05/test-1-37/test-portal-acceptance-pullrequest(master)/22238/playwright-js-tomcat90-postgresql163/4/0/playwright-report/index.html">Playwright Report</a><ol><li><details><summary><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1260567/testReport/batch-planner_main_export.spec/ts">batch-planner/main/export.spec.ts</a></summary><div><ol><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1260567//testReport/batch-planner_main_export.spec/ts/can_export_as_JSON_with_all_field_types_mapped">can export as JSON with all field types mapped</a></li></ol></div></details></li></ol></div></li></ol></details></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest(master)/22238/">test-portal-acceptance-pullrequest(master)#22238</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-37.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/22238/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>headless-playwright<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/ZakiMiklos/liferay-portal/pull/8">ZakiMiklos#8</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357/build/211001511">[master] ci:test:headless-playwright - ZakiMiklos &gt; ZakiMiklos - PR#8 - 2025-05-07[05:47:12]</a><br/></div><div><strong>Testray Build ID:</strong><span></span><code>211001511</code><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-37.liferay.com/job/publish-testray-report/47705/">publish-testray-report#47705</a><br/></div></div>

kevenleone committed:
LPD-54963 Update ListView emptyState

kevenleone committed:
LPD-54963 Update MarketplaceContext

ccorreagg committed:
LPD-54468 Optimize and use existing methods. I think we don't need to pass any content type in general, that is handled internally by JAX-RS. So I remove the new method

algobob committed:
LPD-53553 SF Sorting

ccorreagg committed:
LPD-54468 Use a name more attached to the use case (autogeneration of code for Multipart endpoints)

ccorreagg committed:
LPD-54468 Auto SF

ccorreagg committed:
LPD-54468 buildREST

ChrisKian committed:
LPD-46627 Since we are now expecting user group creation, adjust the first test and remove the two others, since they no longer make sense.

shuyangzhou committed:
LPD-55207 Re-jar workaround for portal-test and util-taglib which do bnd include of jakarta transformed jar. This is a bnd bug that generates broken "CEN Zip64 extra headers". Once devtool upgrades bnd, this workaround should be removed.

kevenleone committed:
LPD-54963 Resize tables

kevenleone committed:
LPD-54963 Add filter

DavysonMelo committed:
LPD-55103 Remove beta FeatureIndicator

DavysonMelo committed:
LPD-55103 Remove frontend-js-components-web from the learn resource context

brittneyq committed:
LRCI-5863 Use correct database type for stable rule

cdbm committed:
LPD-41009 use private class and remove unecessary static declarations

brookedalton committed:
LPD-55047 Update path

pyoo47 committed:
LRCI-5807 Simplify and check for null

pyoo47 committed:
LRCI-5807 Maintain consistency between compareTo and equals

ChrisKian committed:
LPD-46627 Fix test, verify existing user group is fetched and new user group is added

kevenleone committed:
LPD-54963 Prepend ListView cacheKey by the provided ID

brianwulbern committed:
LPD-51861 Update stable

pedro-oliveira446 committed:
LPD-25838 Add model listener to update the theme id when the configuration is changed

marcosapmf committed:
LPD-55009 test fix

brianwulbern committed:
LPD-51861 Auto SF

ambrinchaudhary committed:
LPD-55136 click outside so the Save button is enabled

rafaprax committed:
LPD-54453 DataGuard will delete this

rafaprax committed:
LPD-54453 Reduce the try block to make sure that popServiceContext() will only run if the pushServiceContext happened

brianikim committed:
LPD-53284 Set boolean values properly

rafaprax committed:
LPD-54453 Rename

rachaelkoestartyo committed:
LPD-53971 implement filters

tinatian committed:
LPD-52892 Fix integration tests

brianchandotcom committed:
LRCI-5807 prep next

brianchandotcom committed:
LPD-55006 Rename

brianchandotcom committed:
LPD-55006 The calling method is _downloadAndInstallElasticsearch

brianchandotcom committed:
LPD-55006 Auto SF

brianchandotcom committed:
LPD-54702 Wordsmith

brianchandotcom committed:
LPD-54702 Use interface

brianchandotcom committed:
LPD-41009 As used

brianchandotcom committed:
LPD-55091 SF

magjed4289 committed:
LPD-55141 Add thumbnailURL - due to LPD-53915

magjed4289 committed:
LPD-55141 SF

bryceosterhaus committed:
LPD-53974 - Escape the currentURL value

brianchandotcom committed:
LPD-54533 Wordsmith

brianchandotcom committed:
LPD-54533 Sort

brianchandotcom committed:
LPD-54533 Can we sort this?

brianchandotcom committed:
LPD-54533 SF

brianchandotcom committed:
LPD-54533 Regen

brianchandotcom committed:
LPD-54533 prep next

brianchandotcom committed:
LPD-54533 prep next

brianchandotcom committed:
LPD-53948 What is this weird slash for?

MumenTayyem committed:
LRDOCS-14839 Batch CXs#4

smottal committed:
LPD-54859 Filter specifications with space

smottal committed:
LPD-54859 Integration test

liferay-translation committed:
LPS-77699 Update Translations

marianoalvarosaiz committed:
LPD-52116 Prepare for DB2.

pedro-oliveira446 committed:
LPD-25838 Unit Test

DavysonMelo committed:
LPD-52475 Replace CanSortWithERCField and CanFilterAndSortEntriesViaMetadataFields for playwirght test

daiazius committed:
LPD-52902 Add test to check if the SimpleCaptchaImpl setup methods now use latest configuration

kevenleone committed:
LPD-54963 SF

daiazius committed:
LPD-52902 baseline

brianchandotcom committed:
LPD-54250 SF

brianchandotcom committed:
LPD-54468 SF

abfranca committed:
LRSD-7216 Update permissions for business event object

brianchandotcom committed:
LPD-54468 prep next

brianchandotcom committed:
LPD-54468 prep next

tinatian committed:
LPD-53804 Add ignore tests

tinatian committed:
LPD-53805 portlet-tck-bridge: setup cookies in every startup, insert cookie names to "_internalCookies" to avoid additional logs from CookiesManager, and log the cookie names added.

michaelhashimoto committed:
LRCI-5870 Remove unnecessary verbosity when extracting from archives

rafaprax committed:
LPD-46627 SF

daiazius committed:
LPD-52902 SF Inline, rename

daiazius committed:
LPD-52902 SF just return the captcha provider directly to eliminate field

rafaprax committed:
LPD-52902 Extract repetitive logic to a helper method

rafaprax committed:
LPD-52902 Pass CaptchaConfiguration via method parameter instead

kevhlee committed:
LPD-55217 Fix compile error in SerializerTest.java

The variable 'bufferQueue' was being redefined in the method 'testReleaseLargeBuffer()',
causing compilation to break for the 'com.liferay.petra.io' tests. A variable rename is
necessary to fix this issue. This issue was caused by LPD-44842, specifically commit
58c8439cbeb08246d19397f130f0e2164ef49ea4.

daiazius committed:
LPD-52902 SF Fix test, convert to field and reuse

brianchandotcom committed:
LPD-55206 SF

brianchandotcom committed:
LPD-53971 SF

brianchandotcom committed:
LPD-46627 SF

brianchandotcom committed:
LPD-46627 auto SF

brianchandotcom committed:
LPD-55207 Document it

ZakiMiklos commented:
ci:test:headless-playwright

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>headless-playwright</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest(master)/14479/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-40.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/14479/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:x: ci:test:headless-playwright - 7 out of 10 jobs passed in 1 hour 10 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/548d73d7d987620487c800305fa7d1da7a96e669">548d73d7d987620487c800305fa7d1da7a96e669</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/1e733cf2f3389a9af3ea5ec9d64c0b88fb17c2cf">1e733cf2f3389a9af3ea5ec9d64c0b88fb17c2cf</a><br/>Jenkins Build URL: <a href="https://testray.liferay.com/#/project/35392/routines/590307/build/212445054">EE Development Acceptance (master) - 1350 - 2025-05-08[20:34:16]</a></p><details><summary><strong>ci:test:headless-playwright - 7 out of 10 jobs PASSED</strong></summary><div><h4>3 Failed Jobs:</h4><ul><li><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest(master)/14479/"><strike>test-portal-acceptance-pullrequest(master)</strike></a></li><li>playwright-js-tomcat90-postgresql163<ul><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1158111/">playwright-js-tomcat90-postgresql163/0/0</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1343619/"><strike>playwright-js-tomcat90-postgresql163/1/0</strike></a></li><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1169970/">playwright-js-tomcat90-postgresql163/2/0</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1243833/">playwright-js-tomcat90-postgresql163/3/0</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1235729/"><strike>playwright-js-tomcat90-postgresql163/4/0</strike></a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1158112/">playwright-js-tomcat90-postgresql163/5/0</a></li><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1065919/">playwright-js-tomcat90-postgresql163/6/0</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1243832/">playwright-js-tomcat90-postgresql163/7/0</a></li><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1065918/">playwright-js-tomcat90-postgresql163/8/0</a></li></ul></li></ul></div><details><summary><strong>7 Successful Jobs:</strong></summary><ul/></details></details><h5>For more details click <a href="https://test-1-40.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/14479/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1343619//consoleText">playwright-js-tomcat90-postgresql163/1/0</a> - <a href="https://storage.cloud.google.com/testray-results/2025-05/test-1-40/test-portal-acceptance-pullrequest(master)/14479/playwright-js-tomcat90-postgresql163/1/0/playwright-report/index.html">Playwright Report</a><ol><li><details><summary><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1343619/testReport/export-import-web_main_stagingUseCase.spec/ts">export-import-web/main/stagingUseCase.spec.ts</a></summary><div><ol><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1343619//testReport/export-import-web_main_stagingUseCase.spec/ts/Staging_publish_template_smoke">Staging publish template smoke</a></li></ol></div></details></li></ol></div></li></ol><hr/><details><summary><strong>Failures in common with <a href="https://test-1-40.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">acceptance upstream results</a> at 1e733cf2f3389a9af3ea5ec9d64c0b88fb17c2cf:</strong></summary><ol><li><div><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1235729//consoleText">playwright-js-tomcat90-postgresql163/4/0</a> - <a href="https://storage.cloud.google.com/testray-results/2025-05/test-1-40/test-portal-acceptance-pullrequest(master)/14479/playwright-js-tomcat90-postgresql163/4/0/playwright-report/index.html">Playwright Report</a><ol><li><details><summary><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1235729/testReport/batch-planner_main_export.spec/ts">batch-planner/main/export.spec.ts</a></summary><div><ol><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1235729//testReport/batch-planner_main_export.spec/ts/can_export_as_JSON_with_all_field_types_mapped">can export as JSON with all field types mapped</a></li></ol></div></details></li></ol></div></li></ol></details></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest(master)/14479/">test-portal-acceptance-pullrequest(master)#14479</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-40.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/14479/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>headless-playwright<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/ZakiMiklos/liferay-portal/pull/8">ZakiMiklos#8</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357/build/212702788">[master] ci:test:headless-playwright - ZakiMiklos &gt; ZakiMiklos - PR#8 - 2025-05-09[03:00:36]</a><br/></div><div><strong>Testray Build ID:</strong><span></span><code>212702788</code><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-40.liferay.com/job/publish-testray-report/31315/">publish-testray-report#31315</a><br/></div></div>

ZakiMiklos committed:
LPD-52573 Add a small wait for staging to be loaded.

ZakiMiklos committed:
LPD-52573 Flakyness fix

ZakiMiklos committed:
LPD-52573 add webcontent content check

ZakiMiklos committed:
LPD-52573 remove mid visibility check

jaimelr10 committed:
LPD-52573 Inline

jaimelr10 committed:
LPD-52573 Make test names lowercase
